src/NServiceBus.Persistence.AzureStorage/obsoletes.cs

```diff
@@ -0,0 +1,75 @@
+﻿#pragma warning disable 1591
+
+namespace NServiceBus
+{
+    using System;
+
+    [ObsoleteEx(Message = "Azure Storage Queues and Azure Service Bus supports timeouts natively and does not require timeout persistence.",
```

Comment:
ASQ hasn't even used timeout persistence in a while. I'd rather have a generic statement saying, Azure transports do not require it. No other transport would use this persistence. How about the following:

```suggestion
    [ObsoleteEx(Message = "Azure transports support timeouts natively and do not require timeout persistence.",
```